README.md

```diff
@@ -370,6 +375,12 @@ defmodule MyModule do
   end
 end
 ```
+<!-- MDOC !-->
+
+## [Contributing](CONTRIBUTING.md)
+
+Before contributing to this project, please read the
+[CONTRIBUTING.md](CONTRIBUTING.md).
```

Comment:
This is duplicated, maybe due to parallel changes I’ve made on the file. Rebasing should help.

Reply:
`README.md` from `docfix/update-docs-and-ref-domo` branch contains the following content?

```
  397 ## [Contributing](CONTRIBUTING.md)
  398 
  399 Before contributing to this project, please read the
  400 [CONTRIBUTING.md](CONTRIBUTING.md).
```

Reply:
`README.md` from `docfix/update-docs-and-ref-domo` branch contains the following content?

```
  397 ## [Contributing](CONTRIBUTING.md)
  398 
  399 Before contributing to this project, please read the
  400 [CONTRIBUTING.md](CONTRIBUTING.md).
```

Reply:
Yes, I’ve changed the order of the sections at the end of the `README.md`. When rebasing, I think you’ve resolved the conflict by duplicating the *Contributing* section. Could you please remove the duplicate?